custom/icds_reports/models/util.py

```diff
@@ -93,6 +93,72 @@ def run_aggregation_queries(self) -> bool:
         return False
 
 
+UCR_MAPPING = {
+    "XFormInstance": {
+        "static-usage_forms": [
+            'http://openrosa.org/formdesigner/F1B73934-8B70-4CEE-B462-3E4C81F80E4A',
+            'http://openrosa.org/formdesigner/2864010F-B1B1-4711-8C59-D5B2B81D65DB',
+            'http://openrosa.org/formdesigner/376FA2E1-6FD1-4C9E-ACB4-E046038CD5E2',
+            'http://openrosa.org/formdesigner/D4A7ABD2-A7B8-431B-A88B-38245173B0AE',
+            'http://openrosa.org/formdesigner/89097FB1-6C08-48BA-95B2-67BCF0C5091D',
+            'http://openrosa.org/formdesigner/792DAF2B-E117-424A-A673-34E1513ABD88',
+            'http://openrosa.org/formdesigner/619B942A-362E-43DE-8650-ED37026D9AC4',
+            'http://openrosa.org/formdesigner/1D568275-1D19-46DB-8C54-2C9765DF6335',
+            'http://openrosa.org/formdesigner/362f76b242d0cfdcec66776f9586dc3620e9cce5',
+            'http://openrosa.org/formdesigner/756ec44475658f3f463f8012632def2bc9fbe731',
+            'http://openrosa.org/formdesigner/66d52f84d606567ea29d5fae88f569d2763b8b62',
+            'http://openrosa.org/formdesigner/b183124a25f2a0ceab266e4564d3526199ac4d75',
+        ],
+        "static-dashboard_growth_monitoring_forms": [
+            'http://openrosa.org/formdesigner/7a55754119359466b1951d7251068bd4f45e73c3',
+            'http://openrosa.org/formdesigner/b183124a25f2a0ceab266e4564d3526199ac4d75',
+            'http://openrosa.org/formdesigner/376FA2E1-6FD1-4C9E-ACB4-E046038CD5E2',
```

Comment:
These 2 were added in this PR

Reply:
The form that Cal mentioned in the ticket falls under this previously missing xmlns https://dimagi-dev.atlassian.net/browse/IIO-827